config/crd/bases/activemonitor.keikoproj.io_healthchecks.yaml

```diff
@@ -53,23 +49,230 @@ spec:
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
           type: string
         metadata:
+          properties:
```

Comment:
metadata can be an object we donot necesarily have to qualify all the fields with latest kubebuilder version. it can be:
       metadata:
          type: object

Reply:
reverted these changes.